### PR TITLE
@vue/cli: init at 3.0.1

### PR DESCRIPTION
**Pull request by tiramiseb**
###### Motivation for this change

Vue CLI v3 stable has been released.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Some other Node packages have been upgraded when executing `generate.sh`. I guess it is okay to include these changes in the PR (because well, It's hard to make sure what is needed by Vue CLI and what has just been upgraded because a new version exists), but I'm not 100% sure.

### CODE REVIEW

File: pkgs/development/node-packages/node-packages-v8.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
 , "sloc"
 , "vue-cli"
 , "swagger"
+, "@vue/cli"
```
[the remark this answers is not in the document]
Comment (matthewbauer) [reply pkgs/development/node-packages/node-packages-v8.json]: Yeah I agree on that. Maybe move everything listed in node-packages-v6.json to node-packages-v10.json so we don't lose anything. My understanding is that node versions are close enough to not be a huge issue for most use cases.

File: pkgs/development/node-packages/node-packages-v8.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
 , "sloc"
 , "vue-cli"
```
Comment (Mic92) [inline pkgs/development/node-packages/node-packages-v8.json]: So `vue-cli` is the old `@vue/cli`?

File: pkgs/development/node-packages/node-packages-v8.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mic92) [reply pkgs/development/node-packages/node-packages-v8.json]: Here it says we should stick to the latest LTS: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/node.section.md which is 8_x

File: pkgs/development/node-packages/node-packages-v8.json (hunk 2 above)
Comment (tiramiseb) [reply pkgs/development/node-packages/node-packages-v8.json]: @vue/cli is Vue CLI v3
vue-cli is Vue CLI v2

Even though v2 is deprecated, most users still use it, we must give them enough time to switch...

File: pkgs/development/node-packages/node-packages-v8.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (adisbladis) [reply pkgs/development/node-packages/node-packages-v8.json]: @Mic92 v6 is still supported by upstream https://github.com/nodejs/Release#release-schedule

[the remark this answers is not in the document]
Comment (Mic92) [reply pkgs/development/node-packages/node-packages-v8.json]: April 2019 means that we need to upgrade before the next release.

File: pkgs/development/node-packages/node-packages-v8.json (hunk 2 above)
Comment (Mic92) [reply pkgs/development/node-packages/node-packages-v8.json]: ok. is it possible to add it to `node-packages-v6`?

Comment (tiramiseb) [reply pkgs/development/node-packages/node-packages-v8.json]: From the vue-cli v3 documentation: "Vue CLI requires Node.js version 8.9"

File: pkgs/development/node-packages/node-packages-v8.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mic92) [reply pkgs/development/node-packages/node-packages-v8.json]: v8 upgrade... https://github.com/NixOS/nixpkgs/pull/45700

**Comment by Mic92** [comment]
This pull request will be part of https://github.com/NixOS/nixpkgs/pull/45700

**Pull request closed without merging**